src/libopensc/muscle.h

```diff
@@ -32,8 +32,8 @@
 #define MSC_MAX_PIN_COMMAND_LENGTH ((1 + MSC_MAX_PIN_LENGTH) * 2)
 
 /* Currently max size handled by muscle driver is 255 ... */
-#define MSC_MAX_READ (card->max_recv_size > 0 ? card->max_recv_size : 255)
-#define MSC_MAX_SEND (card->max_send_size > 0 ? card->max_send_size : 255)
+#define MSC_MAX_READ (MIN(card->max_recv_size,255))
+#define MSC_MAX_SEND (MIN(card->max_send_size,255))
```

Comment:
This is wrong, too. if card->max_recv_size/card->max_send_size is 0, we want it to be 255.

Reply:
Ok, but the card->max_send_size was setting to 255 and card->recv_size was setting to 256, and muscle max is 255..so in ```
muscle_init()``` i initialized both with 255, is that ok?